status-im/status-react
# 🎨 NFT Support via OpenSea

fixes #12435 

### Summary
NFTs are coming to a Status Wallet near you 🥁 🥁 🥁 . This PR adds the ability to see NFTs under the wallet collectibles section. Clicking on an NFT reveals its traits and links to OpenSea.

Missing features:
- Setting up NFTs as PFPs
- Transferring NFTs 
- Sharing NFT links via OS share dialog
- Privacy opt-in similar to link unfurling

### Testing notes
- Try all variations of creating addresses and make sure that NFTs are loaded.
- Pull to refresh should re-fetch NFTs 

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
- Wallet

##### Functional
- wallet / transactions

status: ready
